api/envoy/service/auth/v3/attribute_context.proto

```diff
@@ -144,7 +144,17 @@ message AttributeContext {
     string protocol = 10;
 
     // The HTTP request body.
-    string body = 11;
+    string body = 11 [deprecated = true];
+
+    // The HTTP request body.
+    oneof request_body {
+      // The HTTP request body in bytes.
+      bytes raw_body = 12;
+
+      // [#not-implemented-hide:] The HTTP request body in UTF-8 string.
+      // [#comment:TODO(dio): Use this instead of field body to set request body with UTF-8 string].
```

Comment:
It's not clear to me how we will ever transition over to this, especially with the possible perf hit of dual setting the field. Why not just add a new raw_body field and then have documentation that specifies that either body or raw_body will be set based on the body options?

Reply:
Thanks for this. I was in doubt on how to correctly expressing "one-of". Now it is clear.